app/app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
```

Comment:
you could add a link or button to navigate to the next page, otherwise have to check in app.rb whats the route